hikey.mk

```diff
@@ -32,38 +47,34 @@ EDK2_BIN 			?= $(EDK2_PATH)/Build/HiKey/RELEASE_GCC49/FV/BL33_AP_UEFI.fd
 EDK2_BUILD			?= RELEASE
 endif
 
-LINUX_CONFIG_ADDLIST		?= $(LINUX_PATH)/kernel.config
-
-MCUIMAGE_BIN			?=$(ROOT)/out/mcuimage.bin
-USBNETSH_PATH			?=$(ROOT)/out/usbnet.sh
+MCUIMAGE_BIN			?=$(EDK2_PATH)/HisiPkg/HiKeyPkg/NonFree/mcuimage.bin
 STRACE_PATH			?=$(ROOT)/strace
 BOOT_IMG			?=$(ROOT)/out/boot-fat.uefi.img
 LLOADER_PATH			?=$(ROOT)/l-loader
 NVME_IMG			?=$(ROOT)/out/nvme.img
+OUT_PATH			?=$(ROOT)/out
+GRUB_PATH			?=$(ROOT)/grub
+PATCHES_PATH			?=$(ROOT)/patches_hikey
```

Comment:
Why yet another Git tree? Add this to `$(ROOT)/build/hikey` maybe? (not sure it makes sense in the repo model)